release.nix

```diff
@@ -18,8 +18,6 @@ let
               text-render =
                 haskellPackagesNew.callPackage ./nix/text-render.nix { };
 
-              hnix = haskellPackagesOld.hnix_0_4_0;
-
```

Comment:
This is unrelated but problematic for the version of nixpkgs I'm using. I'll address this somehow...